test("First Test", () => {
  throw new Error("Algo deu errado");
});
